### DISCUSSION TITLE
部署rsshub遇到的问题

### DISCUSSION THREAD

**Discussion by faustliang** [DIYgod/RSSHub#9507] [Q&A]
![image](https://user-images.githubusercontent.com/63723974/162610996-bfcc794b-e22e-496b-8120-f198c23b169e.png)
![image](https://user-images.githubusercontent.com/63723974/162611043-bff7ba96-4ba7-4eeb-bdeb-9d17b9120dbd.png)

根据流程搭建，试错很多次，不知道这个哪里错了？

**Comment by TonyRL** [comment]
You need to install nginx

**Comment by Kiotlin** [comment]
当你试图在任何一台服务器上部署网站时，请确保你的服务器至少已经安装了一种[web server](https://zh.wikipedia.org/zh-my/%E7%B6%B2%E9%A0%81%E4%BC%BA%E6%9C%8D%E5%99%A8)，如[Nginx](https://nginx.org/en/docs/beginners_guide.html)，并保持后台运行，找到网站根目录，按照[教程](https://docs.rsshub.app/install/#shou-dong-bu-shu)安装，运行。注意保持nginx配置文件中的网站根目录和rsshub文件根目录一致。
